compiler-rt/lib/scudo/standalone/memtag.h

```diff
@@ -269,7 +274,7 @@ inline NORETURN void enableSystemMemoryTaggingTestOnly() {
 }
 
 struct ScopedDisableMemoryTagChecks {
-  ScopedDisableMemoryTagChecks() {}
+  ScopedDisableMemoryTagChecks(bool cond UNUSED = true) {}
```

Comment:
```suggestion
  ScopedDisableMemoryTagChecks(UNUSED bool cond) {}
```

Reply:
doesn't this mean we no longer can initiate without the argument?

Reply:
sorry my bad, just reorder the `UNUSED`

Reply:
applied